### PR TITLE
[network] Support netstat in Windows

### PR DESCRIPTION
**Pull request by mausch**
Implements #2350

### CODE REVIEW

**Comment by mausch** [comment]
Is the Travis build working at all? The build log seems to have lots of errors unrelated to my changes.

**Comment by olivielpeau** [comment]
Thanks @mausch for this addition!

We've had a lot of errors on Python 2.6 lately but we're in the process of removing support of Python 2.6. The tests related to your changes are passing.

We'll spend some time soon to review and test your PR.

File: checks.d/network.py [hunk 1]
```diff
@@ -508,3 +526,35 @@ def _parse_solaris_netstat(self, netstat_output):
...
 
         return metrics_by_interface
+
+    def _check_windows(self, instance):
+        output, _, _ = get_subprocess_output(['netstat', '-an'], self.log)
+        return self._parse_windows_netstat(output)
+
+    def _parse_windows_netstat(self, netstat):
+        state_map = Network.TCP_STATES['netstat_windows']
+        lines = netstat.split('\n')
```
Comment (remh) [inline checks.d/network.py]: you should use `netstat.splitlines()` here instead.

**Comment by remh** [comment]
Looks good besides the nitpick, could you rebase against the master branch and also squash your commits please ?

**Comment by masci** [comment]
Going to finish this on #2478 b/c 5.8 is approaching fast, thanks again @mausch!

**Pull request closed without merging**